src/PairEvaluatorColloid.h

```diff
@@ -34,44 +66,35 @@ namespace detail
  *
  * The pair potential has three different coupling styles between particle types:
  *
- * - ``slv-slv`` gives the Lennard-Jones potential for coupling between pointlike particles,
- *   with the standard \f$4 \varepsilon\f$ replaced by \f$A/36\f$.
- * - ``coll-slv`` gives the interaction between a pointlike particle and a colloid
- * - ``coll-coll`` gives the interaction between two colloids
+ * - Solvent-Solvent gives the Lennard-Jones potential for coupling between pointlike particles,
+ *   with the standard \f$4 \varhamaker\f$ replaced by \f$A/36\f$.
+ * - Colloid-Solvent gives the interaction between a pointlike particle and a colloid
+ * - Colloid-Colloid gives the interaction between two colloids
  *
  * Refer to the work by <a href="http://dx.doi.org/10.1063/1.3578181">Grest et al.</a> for the
  * form of the colloid-solvent and colloid-colloid potentials, which are too cumbersome
  * to report on here.
  *
  * \warning
- * The ``coll-slv`` and ``coll-coll`` styles make use of the particle diameters to
- * compute the interactions. In the ``coll-slv`` case, the identity of the colloid
- * in the pair is inferred to be the larger of the two diameters. You must make
- * sure you appropriately set the particle diameters in the particle data.
+ * The diameter parameters are used to infer which case is used to compute
+ * the interactions. A particle diameter equal to 0 is used to infer
+ * a solvent interaction. You must make sure you appropriately set the diameter.
  *
  * The strength of all potentials is set by the Hamaker constant, represented here by the
  * symbol \f$A\f$. The other parameter \f$\sigma\f$ is the diameter of the particles that
  * are integrated out (colloids are comprised of Lennard-Jones particles with parameter
  * \f$\sigma\f$). The parameters that are fed in are:
  *
  * - \a A - the Hamaker constant
+ * - \a a_1 - diameter of particle 1
+ * - \a a_2 - diameter of particle 2
  * - \a sigma_3 - \f$\sigma^3\f$
```

Comment:
I think this discussion can be removed. Some of it isn't quite right anymore, and I would prefer just to keep the Python docs pristine.